vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/View.java

```diff
@@ -28,33 +42,62 @@ public View() {
         this.zoom = 0;
     }
 
+    /**
+     * This method is used for internal synchronization of map configuration.
+     */
     @Override
     public String getType() {
         return Constants.OL_VIEW;
     }
 
+    /**
+     * Gets center coordinates of the map in epsg:3857 format
+     * @return center of the map in epsg:3857 format
+     */
     public Coordinate getCenter() {
         return center;
     }
 
+    /**
+     * Sets the center of the map
```

Comment:
The coordinates specify the center of the viewport, not of the map. The center of the map is a different concept, so it's important to not get these mixed up.

Similarly the other JavaDoc should refer to the viewport, as well.

Reply:
Done.

Reply:
Done.